library/src/main/java/com/devbrackets/android/recyclerext/adapter/RecyclerReorderCursorAdapter.java

```diff
@@ -175,6 +176,17 @@ public SparseIntArray getPositionMap() {
     }
 
     /**
+     * Notifies the adapter that one item was moved from one position to another.
+     * This method can be overridden by subclasses that use {@link ItemTouchHelper} as a way of
+     * interacting with the {@link RecyclerView} elements to call {@link #notifyItemMoved(int, int)}.
+     * @param originalPosition The original position of the item
+     * @param newPosition The new position of the item
+     */
+    protected void notifyItemReordered(int originalPosition, int newPosition) {
+        notifyDataSetChanged();
```

Comment:
It's been a while since I worked in this area; is there any downside to calling 
```java
notifyItemChanged(originalPosition);
notifyItemChanged(newPosition);
```